lib/Store/CouchDB.pm

```diff
@@ -805,7 +820,7 @@ sub list_view {
 
     $self->method('GET');
 
-    return $self->_call($path);
+    return $self->_call($path, 1);
```

Comment:
would be cool to use `'accept_stale'` as a string and then test for truthiness in `_call`.

Having `1` as a parameter is a bit confusing. Intuitively I'd guess it was a timeout or number of retries rather than a option that's specific to calling views.